scripts/preflight-dev-ports.cjs

```
@@ -0,0 +1,382 @@
+          getProcessInfo: getProcessInfoWindows,
+          listNodeProcesses: listNodeProcessesWindows,
+          kill: (pid) =>
+              run('taskkill', ['/PID', String(pid), '/T', '/F']).status === 0,
+      }
+    : {
+          listListeningPids: findListeningPidsUnix,
+          getProcessInfo: getProcessInfoUnix,
+          listNodeProcesses: listNodeProcessesUnix,
+          kill: (pid) => {
+              try {
+                  process.kill(pid, 'SIGTERM');
```

ESLint

[error] 'process' is not defined.

(no-undef)